### PR TITLE
qt-gui 0.0.7

### PR DESCRIPTION
**Pull request by 0003088**
Addressed current issues.

### CODE REVIEW

File: src/tools/qt-gui/qml/main.qml [hunk 1]
```diff
@@ -654,9 +654,9 @@ ApplicationWindow {
 					alternatingRowColors: false
 					backgroundVisible: false
 
-					Component.onCompleted: treeView.updateIndicator.connect(updateModel)
+//					Component.onCompleted: treeView.updateIndicator.connect(updateModel)
 
-					onUpdateModel: model = getModel()
+//					onUpdateModel: model = getModel()
```
Comment (markus2330) [inline src/tools/qt-gui/qml/main.qml]: Why not remove the lines?

File: src/tools/qt-gui/qml/main.qml [hunk 2]
```diff
@@ -40,7 +40,7 @@ ApplicationWindow {
 	property bool	error: false
 	property bool	helpMode: false
 
-	property string version: "0.0.6 (beta)"
+	property string version: "0.0.7 (beta)"
```
Comment (markus2330) [inline src/tools/qt-gui/qml/main.qml]: Still beta ;) ?

File: src/tools/qt-gui/src/treeviewmodel.cpp [hunk 3]
```diff
@@ -191,9 +204,8 @@ void TreeViewModel::importConfiguration(const QString& name, const QString& form
 	{
 		result = merger.mergeKeySet(MergeTask (BaseMergeKeys (base, root), OurMergeKeys (base, root), TheirMergeKeys (importedKeys, root), root));
 	}
-	catch(...)//TODO
+	catch(...)//TODO: Which exceptions are possible?
```
Comment (markus2330) [inline src/tools/qt-gui/src/treeviewmodel.cpp]: Please open a github question. It should be documented, but isnt.

File: src/tools/qt-gui/src/treeviewmodel.cpp [hunk 4]
```diff
@@ -191,9 +204,8 @@ void TreeViewModel::importConfiguration(const QString& name, const QString& form
...
 	{
-		qDebug() << "Could not merge keysets";
 		emit showMessage(tr("Error"), tr("Could not merge keys."),"");
```
Comment (markus2330) [inline src/tools/qt-gui/src/treeviewmodel.cpp]: The example shows how you get information about the conflicts. That should be present in the dialog. (its enough to have it just as text)

File: src/tools/qt-gui/qml/main.qml (hunk 2 above)
Comment (0003088) [inline src/tools/qt-gui/qml/main.qml]: Well, changes to the system branch cannot be saved atm and there are still some crashes, so better play it safe.

File: src/tools/qt-gui/src/treeviewmodel.cpp (hunk 4 above)
Comment (0003088) [inline src/tools/qt-gui/src/treeviewmodel.cpp]: Resolved in c69adf0dec9a858a422962a070f88783a31118e6

File: src/tools/qt-gui/src/treeviewmodel.cpp (hunk 3 above)
Comment (0003088) [inline src/tools/qt-gui/src/treeviewmodel.cpp]: Done

File: src/tools/qt-gui/qml/main.qml (hunk 1 above)
Comment (0003088) [inline src/tools/qt-gui/qml/main.qml]: Done.

**Comment by 0003088** [comment]
The reason for the failed build must be upstream, it builds fine on my machine.

**Comment by markus2330** [comment]
Thank you!

**Pull request merged**